File: Modulo 2/Clase01_2/app/src/main/java/com/area51/clase01_2/AndroidApplication.java
package com.area51.clase01_2;

import android.app.Application;

import io.realm.Realm;
import io.realm.RealmConfiguration;

public class AndroidApplication extends Application {
    @Override
    public void onCreate() {
        super.onCreate();

        Realm.init(this);
        RealmConfiguration configuration =
                new RealmConfiguration.Builder()
                        .schemaVersion(1)
                        .name("clase1")
                        .build();
        Realm.setDefaultConfiguration(configuration);
    }
}

File: Modulo 3/ClaseRecycler/app/src/main/java/com/area51/claserecycler/AndroidApplication.kt
package com.area51.claserecycler

import android.app.Application
import io.realm.Realm
import io.realm.RealmConfiguration

class AndroidApplication : Application() {
    override fun onCreate() {
        super.onCreate()

        Realm.init(this)
        val config = RealmConfiguration.Builder()
            .schemaVersion(1)
            .name("clase.realm")
            .build()
        Realm.setDefaultConfiguration(config)
    }
}